x/authz/module/autocli.go

```diff
@@ -47,7 +49,18 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 			},
 		},
 		Tx: &autocliv1.ServiceCommandDescriptor{
-			Service: authzv1beta1.Msg_ServiceDesc.ServiceName,
+			Service:              authzv1beta1.Msg_ServiceDesc.ServiceName,
+			EnhanceCustomCommand: true,
+			RpcCommandOptions: []*autocliv1.RpcCommandOptions{
+				{
+					RpcMethod:      "Revoke",
+					Use:            "revoke [grantee] [msg-type-url] --from=[granter]",
+					Short:          strings.TrimSpace(`Revoke authorization from a granter to a grantee`),
```

Comment:
Unnecessary to call strings.TrimSpace for a constant string whose start and end we already know.